draft-ietf-tls-esni.md

```diff
@@ -944,13 +944,11 @@ It then computes a string
     accept_confirmation =
         Derive-Secret(Handshake Secret,
                       "ech accept confirmation",
-                      ClientHelloInner..ServerHelloECHConf)
+                      ClientHelloInner || ServerHelloECHConf)
```

Comment:
Actually, the `..` notation is meant to denote the sequence of messages between `ClientHelloInner` and `ServerHelloECHConf`. This sequence is empty here, but I think we should use the notation nonetheless, in order to be consistent with RFC8446.

Reply:
I don’t think this is necessary. || is more clear in this context given that the sequence is always empty.

Reply:
What about incorporating the CH1 hash? Isn't that also something that's implied by the `..` notation?

Reply:
In other words, it denotes the whole transcript from ClientHelloInner to ServerHelloECHConf, including the messages on the HRR path.

Reply:
(@chris-wood and I spoke offline and we agreed that the `..` notation should stay as-is.)

Reply:
I find the range syntax a bit confusing here. If there were messages in between, it's not clear to me which variant of those messages would be selected, or whether some transformation would be applied to them.

I don't feel really strongly about this. Happy to revert it, but to be fully pedantic, I want to point out that RFC 8446 uses three dots between messages. I'll go ahead and use three dots here, unless two dots has a different meaning that I don't understand.

Reply:
Oops, three dots is right! https://tools.ietf.org/html/rfc8446#section-7.1

I think my dyslexic brain is removing  a dot.